.changeset/dirty-bottles-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': patch
+---
+
+Tweak components to use the new border radius values (Anchor, InlineMessage, SearchInput, Selector, Tag).
```

Comment:
Do we need this changeset/does it need to be more specific? There were several small changes but most of them are almost unnoticeable

Reply:
I don' think we need to document changes to the implementation, only if it impacts the public API (developer or user-facing).

Reply:
So I guess that this `patch` changeset is fine, since there is a small visual change but no changes to the public API?

Reply:
I didn't realize there was a visual change. Yeah, in that case I think it's fine.

Reply:
Very small, just border radii, but I think it's still worth mentioning 🙂